src/components/PDFView/PDFPasswordForm.js

```diff
@@ -0,0 +1,83 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import Button from '../Button';
+import Text from '../Text';
+import TextInput from '../TextInput';
+import styles from '../../styles/styles';
+import compose from '../../libs/compose';
+import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../withWindowDimensions';
+
+const propTypes = {
+    /** If the submitted password is invalid (show an error message) */
+    isPasswordInvalid: PropTypes.bool,
+
+    ...withLocalizePropTypes,
+    ...windowDimensionsPropTypes,
+};
+
+const defaultProps = {
+    isPasswordInvalid: false,
+};
+
+class PDFPasswordForm extends Component {
```

Comment:
This has already been discussed. From @Santhosh-Sellavel above:

> I too had a look at pdf view, I believe there's no further refactoring needed for the same.

Reply:
We are still in transition so for now. we can just follow best practices and leave the Form component part.

Reply:
Ok sounds good.

Reply:
I'm looking at `FORMS.md` and it's not clear to me what I need to change. Please note the following:

1. There's no need for validation (on blur) because a PDF password can be anything. There's nothing to validate.
2. It says "Individual form fields should be highlighted with a red error outline". However, this not the case anywhere else in the application. For example,

<img width="322" alt="no_textfield_outline" src="https://user-images.githubusercontent.com/11221215/171966747-b4fa9262-e347-46a4-9ca1-dcb8cc144092.png">

What specifically do you want me to refactor here?

Reply:
- Error styles
- Error on submit. 
- Clear error on Type. 
- Do not disable the button.